code/player/controllers/body_mechanics/AntiStuck.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+using Sandbox;
+
+namespace Cinema;
+
+public partial class AntiStuckMechanic : PlayerBodyControllerMechanic
+{
+    internal int UnstuckAttempts = 0;
+
+    protected override void Simulate()
+    {
+        if( !IsStuck() )
+        {
+            //Reset unstuck attempts if they aren't actually stuck
+            UnstuckAttempts = 0;
+            safePos = Vector3.Zero;
+        }
+        else
```

Comment:
Style: Early return instead of using an `else` statement.
https://github.com/tech-nawar/sbox-cinema/blob/main/STYLE.md#control-statements